core/services/relay/evm/read/event.go

```diff
@@ -406,14 +493,15 @@ func (b *EventBinding) hashTopics(topicTypeID string, topics []any) ([]common.Ha
 		// make topic value for non-fixed bytes array manually because geth MakeTopics doesn't support it
 		topicTyp, exists := b.eventTypes[topicTypeID]
 		if !exists {
-			return nil, fmt.Errorf("cannot find event type entry")
+			return nil, fmt.Errorf("%w: cannot find event type entry for topic: %s", commontypes.ErrNotFound, topicTypeID)
 		}
 
 		if abiArg := topicTyp.Args()[i]; abiArg.Type.T == abi.ArrayTy && (abiArg.Type.Elem != nil && abiArg.Type.Elem.T == abi.UintTy) {
 			packed, err := abi.Arguments{abiArg}.Pack(topic)
 			if err != nil {
-				return nil, err
+				return nil, fmt.Errorf("%w: %s", commontypes.ErrInternal, err.Error())
```

Comment:
abi errors can be confusing, just expand the msg
```suggestion
				return nil, fmt.Errorf("%w: err failed to abi pack topics %s", commontypes.ErrInternal, err.Error())
```